packages/react/src/components/header/components/headerActionBarItem/HeaderActionBarItemWithDropdown.tsx

```diff
@@ -50,6 +50,10 @@ type HeaderActionBarItemWithDropdownProps = React.PropsWithChildren<{
    * Fix the item position to the right side of the action bar.
    */
   fixedRightPosition?: boolean;
+  /**
+   * Render active state in an overlay to prevent button shift when contents change.
+   */
+  showActiveStateInOverlay?: boolean;
```

Comment:
Actually changed to `preventButtonResize`, because in the context of the dropdown `preventResize ` would have not indicated prevent resizing of what? :)